src/metabase/models/field_values.clj

```diff
@@ -1,8 +1,27 @@
 (ns metabase.models.field-values
```

Comment:
I also reorganized this file a bit, so you will see some functions will just be moved around.